project.godot

```diff
@@ -46,3 +46,7 @@ menu={
 "events": [Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":-1,"window_id":0,"alt_pressed":false,"shift_pressed":false,"ctrl_pressed":false,"meta_pressed":false,"pressed":false,"keycode":0,"physical_keycode":4194305,"key_label":0,"unicode":0,"echo":false,"script":null)
 ]
 }
+
+[rendering]
+
+renderer/rendering_method="mobile"
```

Comment:
Whats the reason for this. 

Not familiar with Mobile vs forward vs others

Reply:
More on this here https://docs.godotengine.org/en/stable/contributing/development/core_and_modules/internal_rendering_architecture.html

The TLDR is that it is more performant for what we want to do and is accessible to more devices